analysis_options.yaml

```diff
@@ -27,3 +27,4 @@ linter:
     - unawaited_futures
     - unnecessary_statements
     - unsafe_html
+    - use_super_parameters
```

Comment:
should I add this line?

Reply:
I think so, since it's not part of `flutter_lints`